SettingsList.py

```diff
@@ -5324,6 +5324,35 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             ]
         }
     ),
+    Checkbutton(
+        name           = 'easier_fire_arrow_entry',
+        gui_text       = 'Easier Fire Arrow Entry',
+        gui_tooltip    = '''\
+            It is possible to open the Shadow Temple entrance
+            by lighting the torches with Fire Arrows, but
+            can be difficult to light all 24 torches in time.
+            Enabling this setting allows you to reduce the
+            number of torches that need to be lit to open
+            the entrance, making it easier to perform
+            Fire Arrow Entry.
```

Comment:
The description should explain the lack of logical implications and mention the trick.